tests/features/load_configuration.feature

```diff
@@ -0,0 +1,43 @@
+Feature: Loading values into the configuration
+
+Scenario: Load configuration initialised from the Manifest
+    When I configure the app to run in the "skipBugsnag" state
+    And I run "LoadConfigurationFromManifestScenario"
+    Then I wait to receive a request
+    And the request is valid for the error reporting API version "4.0" for the "Android Bugsnag Notifier" notifier with the apiKey "abc12312312312312312312312312312"
+    And the exception "message" equals "LoadConfigurationFromManifestScenario"
+    And the event "app.releaseStage" equals "testing"
+    And the payload field "events.0.breadcrumbs" is an array with 1 elements
+    And the event "metaData.test.foo" equals "bar"
+    And the event "metaData.test.filter_me" equals "[REDACTED]"
+    And the event "app.versionCode" equals 753
+    #And the event "app.buildUUID" equals "test-7.5.3"
```

Comment:
We should create an internal ticket to enable this again once the gradle plugin fix has been released

Reply:
Can this comment be resolved now?